gateway/api.go

```diff
@@ -979,6 +980,10 @@ func keyHandler(w http.ResponseWriter, r *http.Request) {
 		}
 
 	case http.MethodDelete:
+		if certID != "" {
+			keyName =  generateToken(orgID, certID)
```

Comment:
I'm not quite sure why it requires a special case. Why we can't just standard "key" based on certificate fingerprint from the dashboard side?

Reply:
@buger makes sense. We do not longer need this PR. Made some changes in dashboard to take advantage of key deletion using only orgId+certId